requirements.txt

```diff
@@ -0,0 +1,15 @@
+accelerate>=0.25.0
+draccus
+einops
+# flash_attn>=2.3.3  # Note: As per your comment, this is for documentation and should be installed after an editable install.
+jsonlines
+rich
+timm>=0.9.10
+torch>=2.1.0
+torchvision>=0.16.0
+torchaudio
+transformers>=4.34.1
+wandb
+ninja
+flash-attn
+git+https://github.com/jmercat/open_lm_.git@input_embed
```

Comment:
I also made a PR on open_lm main to avoid this specific branch dependency

Reply:
approved that so after that goes through we can update this